### PR TITLE
UI - Document Required Learn More Link Update

### PR DESCRIPTION
**Pull request by shaangill025**
*Issue:*

- https://github.com/bcgov/entity/issues/23856

*Description of changes:*
Just updating the link.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

File: strr-web/components/bcros/form-section/principal-residence/Form.vue [hunk 1]
```diff
@@ -91,8 +91,7 @@
                 class="text-bcGovColor-activeBlue"
                 target="_blank"
                 :href="`
-                  https://www2.gov.bc.ca/gov/content/housing-tenancy/\
-                    short-term-rentals/principal-residence-requirement
+                  https://www2.gov.bc.ca/gov/content/housing-tenancy/short-term-rentals/principal-residence-requirement
```
Comment (dimak1) [inline strr-web/components/bcros/form-section/principal-residence/Form.vue:94]: If a static link you can remote the `:` and back ticks.

**Review by dimak1 — APPROVED** [review]

File: strr-web/components/bcros/form-section/principal-residence/Form.vue (hunk 1 above)
Comment (shaangill025) [reply strr-web/components/bcros/form-section/principal-residence/Form.vue:94]: Fixed

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-311-j5eobjle.web.app

File: strr-web/components/bcros/form-section/principal-residence/Form.vue (hunk 1 above)
Comment (dimak1) [reply strr-web/components/bcros/form-section/principal-residence/Form.vue:94]: Almost... back ticks ` ` needs to be removed.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-311-j5eobjle.web.app

**Pull request merged**